website/public/main.css

```diff
@@ -532,6 +532,7 @@ kbd.keystroke {
   padding: 5px;
   font-size: 0.8rem;
   max-width: 300px;
+  user-select: none;
```

Comment:
I think this deserves a comment explaining why we want to disable text selection.

Reply:
@strager You will never be able to select the error box if it's to the right of your cursor; this special case is just because you set the error box span and do an animation. We could just set it inline CSS on this specific span :thinking: and undo it in main.css . Up to you..., or I can put a comment...